Update error-templates.md (#2546)
TykTechnologies/tyk-docs: tyk-docs/content/advanced-configuration/error-templates.md

As of v2.3 it is possible to use different templates for specific `HTTP error codes`. The `content-type` header of the request is also checked, enabling the usage of different template formats, e.g. an XML template.

> Please note that it is not possible to override the default message for HTTP 404 errors, which indicate that the requested resource was not available (e.g. the requested URL does not exist).

Reviewer: @andyo-tyk  LGTM. Maybe consider the following suggestion:

```suggestion
Please note that it is not possible to override the default message for HTTP 404 errors. These errors indicate that the requested resource could not be found (e.g. the requested URL does not exist).
```
